src/main/kotlin/no/nav/syfo/db/UtsendtVarselFeiletDAO.kt

```diff
@@ -5,6 +5,30 @@ import no.nav.syfo.domain.PersonIdent
 import java.sql.Timestamp
 import java.util.*
 
+fun DatabaseInterface.fetchUtsendtBrukernotifikasjonVarselFeilet(): List<PUtsendtVarselFeilet> {
+    val queryStatement = """SELECT *
+                            FROM UTSENDING_VARSEL_FEILET feilet
+                            WHERE feilet.KANAL = 'BRUKERNOTIFIKASJON'
+                            AND feilet.UTSENDT_FORSOK_TIDSPUNKT  >= '2025-02-27'
+                            AND feilet.is_resendt = FALSE
+                            AND feilet.hendelsetype_navn in 
+                            ('SM_DIALOGMOTE_SVAR_MOTEBEHOV', 'SM_DIALOGMOTE_INNKALT', 'SM_DIALOGMOTE_AVLYST', 'SM_DIALOGMOTE_NYTT_TID_STED', 'SM_MER_VEILEDNING')
+                            AND feilet.UUID_EKSTERN_REFERANSE NOT IN (
```

Comment:
Denne ekstern referanse uuid-sjekken skal strengt tatt ikke være nødvendig lenger, da jeg har innført is_resendt kolonnen. Men tenkte kanskje bare å beholde den for double safety measure, skal i teorien ikke skade heller.